db/stats.py

```diff
@@ -218,8 +247,8 @@ def load_statistics_data(limit=None):
         ret = []
         for line in stats_result:
```

Comment:
I think that it would be a good idea to also select the statistics cached in redis for the most recent hour and add it to the list of statistics shown on the graph. This way the graph will update hourly throughout the day with the most recent data.
We should do something like the following:
* Get data from redis
* If it doesn't exist, skip it
* If it does exist, and is later than the most recent value in the database, append it